### PR TITLE
Feature/api common

### PR DESCRIPTION
**Pull request by pyc8992**
common api merged

### CODE REVIEW

File: server/routes/hashtag.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
 const express= require('express');
 const router = express.Router();
-const postCtrl = require('../controllers/hashtag');
+const hashtagCtrl = require('../controllers/hashtag');
 
 router.get('/posts', 
-  postCtrl.getPostCountGroupByHashtag);
+  hashtagCtrl.getPostCountGroupByHashtag);
```
Comment (HongJungHee) [inline server/routes/hashtag.js:6]: ggggg

Comment (pyc8992) [reply server/routes/hashtag.js:6]: test

**Review by HongJungHee — APPROVED** [review]
gg

**Review by HongJungHee — APPROVED** [review]
go

**Pull request merged**